crawler/jobs/cherrypicker_test_data.py

```diff
@@ -0,0 +1,154 @@
+from bson.objectid import ObjectId
+from datetime import datetime
+from functools import reduce
+import json
+import logging
+import os
+from typing import List
+
+from crawler.constants import (
+    COLLECTION_CHERRYPICK_TEST_DATA,
+    FIELD_MONGODB_ID,
+    FIELD_UPDATED_AT,
+    FIELD_STATUS,
+    FIELD_PLATE_SPECS,
+    FIELD_BARCODES,
+    FIELD_FAILURE_REASON,
+    FIELD_STATUS_PENDING,
+    FIELD_STATUS_STARTED,
+    FIELD_STATUS_PREPARING_DATA,
+    FIELD_STATUS_CRAWLING_DATA,
+    FIELD_STATUS_COMPLETED,
+    FIELD_STATUS_FAILED,
+    TEST_DATA_CENTRE_PREFIX,
+    TEST_DATA_ERROR_NO_RUN_FOR_ID,
+    TEST_DATA_ERROR_WRONG_STATE,
+    TEST_DATA_ERROR_INVALID_PLATE_SPECS,
+    TEST_DATA_ERROR_NUMBER_OF_PLATES,
+    TEST_DATA_ERROR_NUMBER_OF_POS_SAMPLES,
+)
+from crawler.db.mongo import (
+    create_mongo_client,
+    get_mongo_collection,
+    get_mongo_db,
+)
+from crawler.helpers.cherrypicker_test_data import (
+    create_barcodes,
+    create_barcode_meta,
+    create_csv_rows,
+    write_plates_file,
+)
+from crawler.helpers.general_helpers import get_config
+from crawler.main import run as run_crawler
+from crawler.types import Config
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestDataError(Exception):
+    def __init__(self, message):
+        self.message = message
+
+
+def process(run_id: str, config: Config = None) -> List[List[str]]:
```

Comment:
Is there a check for environment here somewhere? We really must not let this run in production!

Reply:
I've added a config value that can be set to True in UAT only.  The default is False and I've then overridden with True in each of the dev environments.  UAT is specified on the deployment project so that will need to be updated to set this option correctly there.